### PR TITLE
Add clarification in Atoms section in Unicode syntax page

### PR DESCRIPTION
**Pull request by eksperimental**
It addresses https://github.com/elixir-lang/elixir/pull/10723#pullrequestreview-592080497

### CODE REVIEW

File: lib/elixir/pages/unicode-syntax.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ Unicode atoms in Elixir follow the identifier rule above with the following modi
   * `<Start>` additionally includes the code point `_` (005F)
   * `<Continue>` additionally includes the code point `@` (0040)
 
-> Note that all Elixir operators are also valid atoms. Therefore `:+`, `:@`, `:|>`, and others are all valid atoms. Atoms can also be quoted, which allows any character, such as `:"hello world"`. The full description of valid atoms is available in the Syntax Reference, this document covers only the rules for identifier-based atoms.
+This document covers only the rules for identifier-based atoms. Atoms can also be quoted, which allows any characters, such as `:'🌢 Elixir'`. The full description of valid atoms is available in the ["Atoms" section in the syntax reference](syntax-reference.html#atoms).
+
+> Note that all Elixir operators are also valid atoms. Therefore `:+`, `:@`, `:|>`, and others are all valid atoms.
```
Comment (josevalim) [inline lib/elixir/pages/unicode-syntax.md:44]:
```suggestion
Note atoms can also be quoted, which allows any characters, such as `:"hello elixir"`. All Elixir operators are also valid atoms (`:+`, `:@`, `:|>`, etc). The full description of valid atoms is available in the ["Atoms" section in the syntax reference](syntax-reference.html#atoms).
```

Comment (eksperimental) [reply lib/elixir/pages/unicode-syntax.md:44]: I commit your suggestion, but I need to correct something first, so please do not merge it yet

File: lib/elixir/pages/unicode-syntax.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Unicode atoms in Elixir follow the identifier rule above with the following modi
   * `<Start>` additionally includes the code point `_` (005F)
   * `<Continue>` additionally includes the code point `@` (0040)
 
-> Note that all Elixir operators are also valid atoms. Therefore `:+`, `:@`, `:|>`, and others are all valid atoms. Atoms can also be quoted, which allows any character, such as `:"hello world"`. The full description of valid atoms is available in the Syntax Reference, this document covers only the rules for identifier-based atoms.
+Note atoms can also be quoted, which allows any characters, such as `:"hello elixir"`. All Elixir operators are also valid atoms (`:+`, `:@`, `:|>`, etc). The full description of valid atoms is available in the ["Atoms" section in the syntax reference](syntax-reference.html#atoms).
```
Comment (eksperimental) [inline lib/elixir/pages/unicode-syntax.md:42]:
```suggestion
Note atoms can also be quoted, which allows any characters, such as `:"hello elixir"`. All Elixir operators are also valid atoms (`:+`, `:@`, `:|>`, etc.). The full description of valid atoms is available in the ["Atoms" section in the syntax reference](syntax-reference.html#atoms).
```

**Comment by eksperimental** [comment]
Good call on removing the quotation.
This is ready to be merged @josevalim

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**